src/workerd/api/r2-bucket.h

```diff
@@ -394,6 +397,8 @@ class R2Bucket: public jsg::Object {
   FeatureFlags featureFlags;
   uint clientIndex;
   kj::Maybe<kj::String> adminBucket;
+  kj::Maybe<kj::String> adminAccount;
```

Comment:
It's weird to have these be separate things. Why not just `kj::Maybe<kj::String> path`?

Reply:
NVM.